src/Reducer/Reducer.jsx

```diff
@@ -0,0 +1,19 @@
+import React, { useReducer } from "react";
+import { DECREMENT, INCREMENT, initialState } from "../Constants/Actions";
+import { ReducerRoot } from "./ReducerRoot";
+
+const Reducer = () => {
+    const [todos, dispatch] = useReducer(ReducerRoot, initialState);
```

Comment:
ReducerRoot, is a function can start from small case
e.g  reducerRoot as well as we can call it as todoReducer